vaadin-list-box-flow-demo/src/main/java/com/vaadin/flow/component/listbox/demo/ListBoxView.java

```diff
@@ -38,140 +43,166 @@
-        // source-example-heading: Adding components between items
+        // source-example-heading: Disabled item
         ListBox<String> listBox = new ListBox<>();
-        listBox.setItems("Bread", "Butter", "Milk");
-
-        // Adding components to the end:
-        listBox.add(new H3("After all the items"));
-
-        // Adding components after a specific item:
-        listBox.addComponents("Butter", new H3("After butter"));
-
-        // Adding components before a specific item:
-        listBox.prependComponents("Bread", new H3("Before bread"));
+        listBox.setItems("Option one", "Option two", "Option three");
+        listBox.setValue("Option one");
+        listBox.setItemEnabledProvider(item -> !"Option three".equals(item));
         // end-source-example
-        addCard("Adding components between items", listBox)
-                .setId("list-box-with-components-between");
+
+        addCard("Disabled item", listBox);
     }
 
-    private void addItemRenderer() {
+    private void multiSelection() {
         // begin-source-example
-        // source-example-heading: Using item renderer and disabling items
-        ListBox<Item> listBox = new ListBox<>();
-        listBox.setItems(getItems());
-
-        listBox.setRenderer(new ComponentRenderer<>(item -> {
-            Label name = new Label("Item: " + item.getName());
-            Label stock = new Label("In stock: " + item.getStock());
+        // source-example-heading: Multi select list box
+        MultiSelectListBox<String> listBox = new MultiSelectListBox<>();
+        listBox.setItems("Option one", "Option two", "Option three",
+                "Option four");
+        // end-source-example
 
-            NativeButton button = new NativeButton("Buy", event -> {
-                item.setStock(item.getStock() - 1);
-                listBox.getDataProvider().refreshItem(item);
-            });
+        addCard("Multi select list box", listBox);
+    }
 
-            Div labels = new Div(name, stock);
-            Div layout = new Div(labels, button);
+    private void separatorDemo() {
+        // begin-source-example
+        // source-example-heading: Separators
+        ListBox<Weekday> listBox = new ListBox<>();
+        listBox.setItems(Weekday.values());
+        listBox.setValue(Weekday.MONDAY);
+        listBox.addComponents(Weekday.FRIDAY, new Hr());
+        // end-source-example
 
-            labels.getStyle().set("display", "flex")
-                    .set("flexDirection", "column").set("marginRight", "10px");
-            layout.getStyle().set("display", "flex")
-                    .set("alignItems", "center");
+        addCard("Presentation", "Separators", listBox);
+    }
 
-            return layout;
+    private void customOptions() {
+        // begin-source-example
+        // source-example-heading: Customizing the label
+        ListBox<Information> listBox = new ListBox<>();
+        List<Information> list = Arrays.asList(
```

Comment:
Same comment as [here](https://github.com/vaadin/vaadin-radio-button-flow/pull/114#discussion_r345108809). Use something more meaningful as the Bean's name

Reply:
Done.